tests_manual/syscall_module_path/README.txt

```diff
@@ -0,0 +1,52 @@
+Title: module load PATH null records
+
+
+Description:
+This test detects kernels that flood the audit logs with hundreds of
+NULL PATH SYSCALL auxiliary records and the modules that provoke them.
+
+It does this by setting up an audit rule in the config file to log all
+init_module and finit_module syscalls, then once the machine is rebooted
+the test script detects SYSCALL records with NULL PATH records along
+with their provoking modules since the most recent boot.
+
+
+How to run the test:
+
+1) Change to the working directory of the test.
+
+	cd tests_manual/syscall_module_path
+
+2) Set up the test, which adds the tests_manual-syscall_module_path
+   audit rule file to the /etc/audit/rules.d/ audit rule directory to
+   log all init_module and finit_module syscalls.
+
+	./setup
+
```

Comment:
This script rebooted my test machine, which was unexpected considering considering step 3 below.  I think we should try to avoid automatic system shutdown/reboots.

Reply:
Ok, removed reboots from setup scripts.